logfile_schema.py

```diff
@@ -31,6 +31,34 @@
 from time import time, tzset
 
 
+def parse_args():
+    """Parse arguments"""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('file',
+                        help='path to log file.')
+    parser.add_argument('timeshift',
+                        help='go e.g. 15m back in time from now.')
+    parser.add_argument('time_regex',
+                        help='e.g. ([0-9]{2}-?){3} ([0-9]{2}:?){3}')
```

Comment:
Just looked through the code. No capturing groups are needed. Maybe we should train people to use non capturing groups, when they do not need them. ^^
So maybe: `(?:[0-9]{2}-?){3} (?:[0-9]{2}:?){3}')